cpp/tests/transform/row_bit_count_test.cu

```diff
@@ -124,8 +160,12 @@ TYPED_TEST(RowBitCountTyped, Lists)
 
   cudf::table_view t({*col});
   auto result = cudf::row_bit_count(t);
-
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(*expected_sizes, *result);
+
+  auto constexpr segment_length     = 3;
```

Comment:
I don't like that `segmented_bit_count` tests are a part of `row_bit_count` tests. Would not mind them being in the same file, but not in the same individual tests. I'd rather have fewer tests targeted at `segmented_bit_count` corner cases.